grammars/rust.cson

```diff
@@ -322,15 +322,27 @@
   }
   {
     'comment': 'Function call'
-    'match': '\\b([a-zA-Z_][a-zA-Z0-9_]*)\\s*\\('
+    'match': '\\b([A-Za-z][A-Za-z0-9_]*|_[A-Za-z][A-Za-z0-9_]*)\\s*\\('
```

Comment:
It is my fault.
The regex is wrong according to the https://doc.rust-lang.org/reference.html#identifiers
I don't know why I didn't look at it before.
The correct regex is `([A-Za-z][_A-Za-z0-9]*|_[_A-Za-z0-9]+)`.
I am sorry that you have to change your solution.